features/facebookImport/src/model/analyses/report/ministories-status-analysis.js

```diff
@@ -13,11 +13,13 @@ export default class MinistoriesStatusAnalysis {
         this._analysesData = analysesResults.map(
             ({ analysis, status, executionTime }) => {
                 return {
```

Comment:
We should probably turn this into its own class. This return value is becoming all too complex to simply be a hash.

Reply:
What we could turn into a class is `{ analysis, status, executionTime }`. That's the result of running an analysis. We could ask it to return the dictionary that would be addd to the report.